vendor/github.com/openshift/library-go/pkg/operator/staticpod/controller/installer/installer_controller.go

```diff
@@ -167,13 +198,17 @@ func (c *InstallerController) newNodeStateForInstallInProgress(currNodeState *op
 	if err != nil {
 		return nil, err
 	}
-	switch installerPod.Status.Phase {
-	case corev1.PodSucceeded:
```

Comment:
Can we nest below here based on the deploymentID and state returned from the `StaticPodInstallStateFunc`?  I think we probably want different status (or at least log messages) about each of the 6 interesting states.